src/components/common/multi-image-resizer.svelte

```diff
@@ -0,0 +1,168 @@
+<script>
+  import { getContext } from 'svelte';
+  import { stores } from '@sapper/app';
+  import { Button, Dialog, Modal, Divider, Loading } from 'attractions';
+  import { snackbarContextKey } from 'attractions/snackbar';
+  import Cropper from 'svelte-easy-crop';
+  import * as api from 'src/utils/api.js';
+
+  const { session } = stores();
+
+  export let aspectRatio = 1;
+  export let open = false;
+
+  let currentImage = 0;
+  let images = [];
+
+  const crop = { x: 0, y: 0 };
+
+  function saveBounds({ detail }) {
+    images[currentImage].pixels = detail.pixels;
+  }
+
+  function resetCropArea() {
+    crop.x = 0;
+    crop.y = 0;
+  }
+
+  async function repositionCursor() {
+    for (let idx = 0; idx < images.length; ++idx) {
+      if (images[idx].uploadPromise == null) {
+        currentImage = idx;
+        resetCropArea();
+        return;
+      }
+    }
+
+    try {
+      currentImage = null;
+      await Promise.all(images.map(image => image.uploadPromise));
+    } finally {
+      open = false;
+      images = [];
+      currentImage = 0;
+    }
+  }
+
+  function dismissImage(image = null) {
```

Comment:
Why does it default to `null`? If `null` is an acceptable value, it needs to be checked for in the body

Reply:
Oh, I just forgot to remove it